src/shared/components/cancerSummary/CancerSummaryContent.tsx

```diff
@@ -358,8 +365,11 @@ export class CancerSummaryContent extends React.Component<ICancerSummaryContentP
             </div>
         ) : null;
         return (
-            <div>
-                <div role="group" className="btn-group">
+
+                <If condition={this.hasAlterations}>
```

Comment:
this is fine but im jw do you have a heuristic that governs when you use `<If>` and when you use `{ bool ? JSX1 : JSX2 }` or `{ bool && JSX }`